Package.swift

```diff
@@ -21,8 +21,8 @@ let package = Package(
         ),
         .testTarget(
             name: "OnboardingTests",
-            path: "Onboarding/Tests",
-            dependencies: ["Onboarding"]
+            dependencies: ["Onboarding"],
+            path: "Onboarding/Tests"
```

Comment:
Order matters here? Seems odd for named variables (kinda defeats the purpose somewhat I think)